dist/a11y.js

```diff
@@ -215,7 +215,7 @@ return JSON.stringify({
     };
   }),
   screen_reader_classes: captureAndLogError(() => {
-    return document.querySelectorAll('.sr-only, .visually-hidden').length > 0;
+    return document.querySelectorAll('.sr-only, .visually-hidden', '.screen-reader-text', 'element-invisible').length > 0;
```

Comment:
I believe this should be `.element-invisible`, per https://www.drupal.org/node/2022859

Reply:
Oops.. Thanks @joedoleson..

Reply:
Oops.. Thanks @joedoleson..

Reply:
So can we update this?

```suggestion
    return document.querySelectorAll('.sr-only, .visually-hidden', '.screen-reader-text', '.element-invisible').length > 0;
```